log/term/terminal_windows.go

```diff
@@ -8,15 +8,23 @@
 package term
 
 import (
+	"bytes"
+	"encoding/binary"
 	"io"
+	"regexp"
 	"syscall"
 	"unsafe"
+
+	"golang.org/x/sys/windows"
```

Comment:
I would rather avoid this enormous dependency since we're only using a few of its functions and they are not difficult to implement locally.